kubejs/server_scripts/enigmatica/kubejs/base/tags/items/enigmatica/ladder_planks.js

```diff
@@ -0,0 +1,40 @@
+onEvent('item.tags', (event) => {
+    event
+        .get('enigmatica:ladder_planks')
+        .add(/_planks$/)
+        .remove([
+            'botania:livingwood_planks',
```

Comment:
Do all these planks have a corresponding ladder?

Reply:
All except the botania ones. I removed those since they're not part of the minecraft planks tag.

Reply:
Cool 👍